derived_datasources/sdmx-abs.ejs

```diff
@@ -420,7 +420,7 @@
     "measureSelectedInitially": null,
     "blacklist": null,
     "cannotSum": null,
-    "description": "This table counts persons where they were staying on Census Night. It also shows the number of persons who were visiting away from their usual residence on Census Night.\n\n"
+    "description": "This table counts persons where they were staying on Census Night. It also shows the number of persons who were visiting away from their usual residence on Census Night.\r\n\r\n"
```

Comment:
@reginapramesti are the `\r`s here intentionally? if not we should probably use `\n` for consistency.

Reply:
Thanks for pointing this out! I don't remember changing this - might've been a Windows thing. I've removed all the `\r`'s